<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="6" column="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QPushButton" name="image_btn">
        <property name="styleSheet">
         <string notr="true">background-color: #007bff;
border: 0;
border-radius: 5px;
height: 25px;
padding: 10px;</string>
        </property>
        <property name="text">
         <string>Ajouter une image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="login">
                <property name="styleSheet">
                 <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
                </property>
                <property name="placeholderText">
                 <string>login</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="genererPass">
              <property name="styleSheet">
               <string notr="true">background-color: #007bff;
border: 0;
border-radius: 5px;
height: 25px;
padding: 10px;</string>
              </property>
              <property name="text">
               <string>Générer</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="mdp">
                <property name="styleSheet">
                 <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>password</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="prenom">
             <property name="styleSheet">
              <string notr="true">height: 40px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>prénom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="societe">
             <property name="styleSheet">
              <string notr="true">height: 40px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>Nom de la société</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nom">
             <property name="styleSheet">
              <string notr="true">height: 40px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>nom</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="observation">
             <property name="placeholderText">
              <string>votre observation</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Liste noire ?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioOui">
               <property name="text">
                <string>Oui</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioNon">
               <property name="text">
                <string>Non</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>137</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>12</y>
           <width>179</width>
           <height>54</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Passport:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>43</y>
           <width>143</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Cin</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>74</y>
           <width>261</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Permis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_6"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>105</y>
           <width>154</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Ville:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ville">
            <property name="styleSheet">
             <string notr="true">#lineEdit::placeholder {
  color: gray;
  font-style: italic;
}</string>
            </property>
            <property name="text">
             <string>Ville: </string>
            </property>
            <property name="placeholderText">
             <string>Nom de la société</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="0">
          <widget class="QTextEdit" name="adresse">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="placeholderText">
            <string>Adresse client</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDateEdit" name="date_permis"/>
         </item>
         <item row="6" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="passport">
             <property name="styleSheet">
              <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>Numero passport</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cin">
             <property name="styleSheet">
              <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>Numéro CIN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="permis">
             <property name="styleSheet">
              <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>Numero permis </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tel">
             <property name="styleSheet">
              <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>téléphone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="email">
             <property name="styleSheet">
              <string notr="true">height: 50px;
border: 0;
border-bottom: 1px solid rgb(100,100,100)</string>
             </property>
             <property name="placeholderText">
              <string>Email client</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="cancelBtn">
        <property name="styleSheet">
         <string notr="true">background-color: red;
color: white;
font-weight: bold;
border: 0;
border-radius: 4px;
height: 50px;</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="valider_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(71, 212, 212);
border: 0;
border-radius: 4px;
height: 40px;</string>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="image_label_cli">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
border: 1px solid rgb(100,100,100);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Uploader votre image ici</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
